settings/README.md

```diff
@@ -0,0 +1,31 @@
+# Monaco Settings
+
+This example shows how [Dynatrace Settings 2.0] can be configured and managed using Monaco.
+
+Each project in this folder includes a different functionality to show its functionalities.
+
+## Projects
+* [basic-example](./basic-example): This project shows how a basic Settings 2.0 object can be managed using Dynatrace. 
+* [first-last](./first-last): This project shows how [ordered settings objects][Dynatrace Settings 2.0 Ordered] can be defined to be at the very front or back of other settings.
+  The example uses application detection settings to display the example.
+
+## Requirements
+
+- Monaco `v2.22.0+`
+- Dynatrace Platform environment
```

Comment:
This is actually not a requirement, right? Classic also works.
Also, in here, OAuth and token are mentioned, but when looking at it, it looks like both are required, but it's only one.